### ISSUE TITLE
can't connect google calendar

### ISSUE THREAD

**Issue by 2amigos-dev** [gadael/gadael#7]
<img width="1172" alt="Screen Shot 2019-03-28 at 09 12 49" src="https://user-images.githubusercontent.com/16922114/55137669-3fb09180-513a-11e9-845e-3685bc31c696.png">
 when I press connect I get this error:

```
Error: Unknown authentication strategy "google"
    at attempt (/var/www/vhosts/gadael/docroot/node_modules/passport/lib/middleware/authenticate.js:166:37)
    at authenticate (/var/www/vhosts/gadael/docroot/node_modules/passport/lib/middleware/authenticate.js:342:7)
    at exports.login (/var/www/vhosts/gadael/docroot/rest/user/googlecalendar.js:33:7)
    at Layer.handle [as handle_request] (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/layer.js:95:5)
    at next (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/layer.js:95:5)
    at /var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:335:12)
    at next (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:275:10)
    at /var/www/vhosts/gadael/docroot/modules/gadaelMiddleware.js:18:9
    at Layer.handle [as handle_request] (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:317:13)
    at /var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:335:12)
    at next (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:275:10)
    at /var/www/vhosts/gadael/docroot/api/Company.api.js:615:13
    at Layer.handle [as handle_request] (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:317:13)
    at /var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:335:12)
    at next (/var/www/vhosts/gadael/docroot/node_modules/express/lib/router/index.js:275:10)
```

```
root@:~# node -v
v8.11.3
root@:~# nodejs -v
v8.11.3
root@:~# npm -v
5.6.0
root@:~# bower -v
1.8.8
```

**Comment by polo2ro** [comment]
Hi,
thanks for this issue, the authentication code is broken and i have started a PR to fix this, you can test the fix with the `fix-google-calendar` branch.
To use this you need to configure the company document with the appropriate client ID and client secret from google oauth API
```json
"calendar": {
    "google": {
	"enable": true,
  	"clientID": "",
	"clientSecret": ""
   }
}
```
There is no UI in gadael to configure this, you need to do this with mongo client

**Comment by polo2ro** [comment]
fixed by #8

**Comment by Almiriad** [comment]
Could you detail more precisely what configuration needs to be done and where for the connection with Google Calendar ?
